app/src/main/java/br/org/cesar/discordtime/stickysessions/ui/session/SessionActivity.java

```diff
@@ -270,14 +270,14 @@ public void displayError(String message) {
     }
 
     @Override
-    public void startLoadingSession() {
-        mLoadingView.setVisibility(View.VISIBLE);
+    public void startLoading() {
+        mLoading.show();
         mRecyclerView.setVisibility(View.INVISIBLE);
```

Comment:
I think we should avoid making recycler view invisible for every single note user adds. Please check here which type of request we're waiting.